sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaWriteSchemaTransformProvider.java

```diff
@@ -167,8 +182,18 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
                   .withKeySerializer(ByteArraySerializer.class)
                   .withValueSerializer(ByteArraySerializer.class));
 
-      return PCollectionRowTuple.of(
-          "errors", outputTuple.get(ERROR_TAG).setRowSchema(ERROR_SCHEMA));
+      PCollection<Row> postWrite =
+          outputTuple
+              .get(OUTPUT_TAG)
+              .apply("post-write", ParDo.of(new NoOutputDoFn<>()))
```

Comment:
This isn't quite safe because we aren't guaranteed the write itself happened yet, just the byte mapping. For now just emit either the empty or error tuple, but drop a TODO to catch any errors KafkaIO itself throws and also follow its output (once it's updated from PDone.)

Reply:
Done.